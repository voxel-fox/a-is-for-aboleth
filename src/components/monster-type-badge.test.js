import MonsterTypeBadge from "./monster-type-badge";

describe("<MonsterTypeBadge />", () => {
  const svgAttrs = { svgAttr: "svgProp", svgAttr2: "svgProp2" };
  const bgAttrs = { bgAttrs: "bgProp", bgAttrs2: "bgProp2" };
  const iconAttrs = { iconAttrs: "iconProp", iconAttrs2: "iconProp2" };
  const type = "monster-type";
  const mockProps = { svgAttrs, bgAttrs, iconAttrs, type };

  it("renders correctly", () => {
    const wrapper = shallow(<MonsterTypeBadge {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
