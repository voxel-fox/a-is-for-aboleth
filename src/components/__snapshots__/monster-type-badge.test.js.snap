// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MonsterTypeBadge /> renders correctly 1`] = `
<svg
  data-creature-type="monster-type"
  svgAttr="svgProp"
  svgAttr2="svgProp2"
  viewBox="0 0 31.5 31.5"
  x="10.5"
  y="9"
>
  <circle
    bgAttrs="bgProp"
    bgAttrs2="bgProp2"
    cx="15.75"
    cy="15.75"
    fill="#fff"
    r="15.25"
    stroke="#000"
    strokeMiterlimit="10"
  />
  <use
    fill="#000"
    iconAttrs="iconProp"
    iconAttrs2="iconProp2"
    xlinkHref="#monster-type"
  />
</svg>
`;
